core/bin/zksync_server/src/main.rs

```diff
@@ -165,6 +170,12 @@ fn main() -> anyhow::Result<()> {
         secrets,
     )?;
 
+    if opt.only_verify_config {
+        node.build(opt.components.0)?;
+        tracing::info!("Config verification is successful!");
```

Comment:
Note that you use `tracing` before installing observability, so it has no effect.